build.sbt

```diff
@@ -33,11 +30,12 @@ resolvers ++= Seq(
   Resolver.bintrayRepo("webjars", "maven"),
   Resolver.typesafeRepo("releases"),
   Resolver.sbtPluginRepo("releases"),
-  Resolver.sonatypeRepo("releases"),
   Resolver.mavenLocal
 )
```

Comment:
Pretty sure you can remove *all* resolvers here? Specially
```sbt
Resolver.bintrayRepo("webjars", "maven"),
Resolver.typesafeRepo("releases"),
Resolver.sbtPluginRepo("releases"),
```
are not needed anymore